Makefile

```diff
@@ -1,9 +1,9 @@
 PKGS := arch release stream stream/rhcos release/rhcos fedoracoreos
 
 build:
-	for pkg in $(PKGS); do (cd $$pkg && go build -mod=vendor); done
+	set -e; for pkg in $(PKGS); do (cd $$pkg && go build -mod=vendor); done
```

Comment:
Hmm, is a better fix anyway to rework this to use a variable rule instead based on `PKGS`?

Reply:
Good call.  Updated!